### PR TITLE
Adding missing field masks

### PR DESCRIPTION
**Pull request by jlazar**
Fixes fields that are lost when updating a cloud function. References issue: https://github.com/google-github-actions/deploy-cloud-functions/issues/259

### CODE REVIEW

**Review by sethvargo — APPROVED** [review]

**Comment by sethvargo** [comment]
@jlazar can you please sign the Google CLA? https://cla.developers.google.com/

**Comment by jlazar** [comment]
@sethvargo what are the next steps to get this one merged in? thanks

**Comment by sethvargo** [comment]
As soon as CI passes 😄

File: src/client.ts [hunk 1]
```diff
@@ -413,11 +413,13 @@ export class CloudFunctionsClient {
       'ingressSettings',
       'serviceAccountEmail',
       'timeout',
+      'minInstances',
       'maxInstances',
       'eventTrigger.eventType',
       'eventTrigger.resource',
       'eventTrigger.service',
       'labels',
+      'secretEnvironmentVariables'
```
Comment (sethvargo) [inline src/client.ts:422]: @jlazar looks like you need a trailing comma here

**Review by sethvargo — APPROVED** [review]

**Pull request merged**